app/views/accounts/sales.html.erb

```diff
@@ -0,0 +1,23 @@
+<!-- Page rendered when anonymous or unsubscribed sign up -->
+<div class="container">
+  <h1>The best resource for intermediate Ember developers</h1>
+  <p>For just $9.95/month, you get <em>instant</em> <em>on-demand</em> access to every single video in the emberscreencasts library.</p>
+
+  <ul>
+      <li>New up-to-date videos every week.</li>
+      <li>Over 7 hours of fast-paced content.</li>
```

Comment:
While you're updating stuff, could you change this 7 to 10? I've been busy since I wrote this :)